src/server/scripts/Spells/spell_generic.cpp

```diff
@@ -2710,6 +2710,48 @@ class spell_gen_profession_research : public SpellScriptLoader
         }
 };
 
+enum TrinketSpells
+{
+    SPELL_PVP_TRINKET_ALLIANCE  = 97403,
+    SPELL_PVP_TRINKET_HORDE     = 97404
+};
+
```

Comment:
// spellid - spellname

this helps alot when searching specific spells later. But it's more or less a visual suggestion.